api/v1alpha1/backstage_types.go

```diff
@@ -31,7 +30,7 @@ type BackstageSpec struct {
 	Application *Application `json:"application,omitempty"`
 
 	// Raw Runtime Objects configuration. For Advanced scenarios.
-	RawRuntimeConfig RuntimeConfig `json:"rawRuntimeConfig,omitempty"`
+	RawRuntimeConfig string `json:"rawRuntimeConfig,omitempty"`
```

Comment:
So BackstageConfigName and LocalDbConfigName are merged into one configmap?

Reply:
yes, it looks more consistent with default config where we have all caonfiguration in one ConfigMap